.travis.yml

```diff
@@ -1,11 +1,53 @@
+# Tell Travis which branches to build.
+branches:
```

Comment:
I should have removed that right off the bat, but as a suggestion, I can configure it to match any setup you want, if you just tell me how you would like it.